app/_src/mesh/features/openshift-quickstart.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Get started with Red Hat OpenShift and Kong Mesh
+content_type: tutorial
+description: This guide explains how to get started on Kong Mesh with Red Hat OpenShift.
+---
+
+In this guide, you will learn how to get {{site.mesh_product_name}} up and running quickly in [standalone mode](/mesh/{{page.kong_version}}/production/deployment/stand-alone/) on [Red Hat OpenShift](https://www.redhat.com/technologies/cloud-computing/openshift). This tutorial assumes some base-level OpenShift knowledge.
+
+This tutorial doesn't require a license because {{site.mesh_product_name}} can start in evaluation mode, which allows you to have up to five data planes or sidecars. This provides just enough data planes to get comfortable with the product and test it out.
+
+This quickstart tutorial covers:
+
+* How to use the Red Hat Certified {{site.mesh_product_name}} images
+* How to implement the required OpenShift security context constraints (SCCs) for the `kong-mesh` sidecar
+* How to deploy a sample application, `kuma-demo`, on the mesh and validate that the application is working
+* How to use the sample application to test the features of {{site.mesh_product_name}}
+
+## Prerequisites
+* [ROSA CLI installed](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-installing-rosa.html) or another OpenShift 4.12 cluster with the ability to create LoadBalancer type Kubernetes Services
+* [An AWS account with ROSA permissions enabled](https://docs.openshift.com/rosa/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-config-aws-account.html)
+* [kubectl CLI installed](https://kubernetes.io/docs/tasks/tools/)
+* [OpenShift Container Platform CLI installed](https://docs.openshift.com/container-platform/latest/cli_reference/openshift_cli/getting-started-cli.html)
+* [Helm 3 installed](https://helm.sh/docs/intro/install/)
+* Clone the {{site.mesh_product_name}} OpenShift quickstart repository:
+    ```bash
+    git clone https://github.com/Kong/kong-mesh-quickstart-openshift.git
+    ```
+* Clone the `kuma-demo` app:
```

Comment:
Remove Clone `kuma-demo` bullet, because `kuma-demo` that will work on openshift is in the `kong-mesh-quickstart` repo